frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -123,6 +123,12 @@ export function applyStreamlitThemeTemplateLayout(
       },
       zeroline: false,
       automargin: true,
+      rangeselector: {
+        bgcolor: colors.bgColor,
+        bordercolor: getGray30(theme),
+        borderwidth: 1,
+        x: 0,
```

Comment:
nit: have we control on any spacing aspects here? The buttons look quite close to the chart, maybe there is a better default to add a bit more spacing.

Reply:
Hm the spacing is because I modified the height to a smaller size. normally it looks like this:
![image](https://user-images.githubusercontent.com/16749069/205371429-d4443e98-d88d-4d22-bde8-6f3a939d9e3e.png)